src/commands/ext-configure.ts

```diff
@@ -75,9 +75,8 @@ export const command = new Command("ext:configure <extensionInstanceId>")
       instanceId,
       projectDir: config.projectDir,
     });
-
     const [immutableParams, tbdParams] = partition(
-      spec.params.concat(spec.systemParams ?? []),
+      (spec.params ?? []).concat(spec.systemParams ?? []),
```

Comment:
Do we need these checks here if we are checking it at the api level?

Reply:
Not really - this is just some extra defensive null checking in case we reintroduce this in the future accidentally. I have a low priority goal to make ext:install and ext:configure work offline for local extensions, and doing so could skip over the code in extensionsApi that ensures these are populated.